pwnlib/shellcraft/templates/mips/linux/cat.asm

```diff
@@ -11,8 +11,8 @@ Example:
 
     >>> f = tempfile.mktemp()
     >>> write(f, 'FLAG')
-    >>> asm  = shellcraft.mips.linux.cat(f)
-    >>> asm += shellcraft.mips.linux.exit(0)
+    >>> sc = shellcraft.mips.linux.cat(f)
+    >>> sc += shellcraft.mips.linux.exit(0)
     >>> run_assembly(asm).recvall()
```

Comment:
Uhm it should probably be changed here too
```suggestion
    >>> run_assembly(sc).recvall()
```

Reply:
This was indeed addressed by a later commit and is already fixed